DeveloperConsole.cs

```diff
@@ -170,8 +203,45 @@ private void CommandRunner(string command)
                             case "exhaust":
                                 result = $"Player {args[1]}'s Exhaustion: {LC_Lib.GetExhaustion(Player)}";
                                 break;
+                            // Returns the player's max insanity
+                            case "max_insanity":
+                                result = $"Player {args[1]}'s Max Insanity: {LC_Lib.GetMaxInsanity(Player)}";
+                                break;
+                            // Returns the player's minimum velocity to take damage
+                            case "min_velocity_to_take_damage":
+                                result = $"Player {args[1]}'s Minimum Velocity To Take Damage: {LC_Lib.GetMinVelocityToTakeDamage(Player)}";
+                                break;
+                            // Returns the player's level
+                            case "level":
+                                result = $"Player {args[1]}'s Level: {LC_Lib.GetLevelNumber(Player)}";
+                                break;
+                        }
+                    }
+                    break;
+
+                // heal_player <player> <value>
+                case "heal_player":
```

Comment:
This is redundant as set health already exists, please remove.

Reply:
It's funny that as soon as I saw this, I realized the exact same thing